setup.cfg

```diff
@@ -1,5 +1,9 @@
 [pytest]
-addopts = -v --cov zdict --cov-report html --cov-report term
+flake8-ignore =
+    zdict/__init__.py ALL
+    zdict/dictionaries/template.py ALL
+
+addopts = --flake8 -v --cov zdict --cov-report html --cov-report term
```

Comment:
I setup a different build for style checking. Please checkout [travis ci env var](https://travis-ci.org/zdict/zdict/builds/120416912).

I want the other build for unit testing.
Only one build for style checking.